src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ChoiceWatchQueueDecisionHandler.java

```diff
@@ -94,7 +94,7 @@ protected void onQueueing(Task p, List<Action> actions)
                 ExtensibleChoiceParameterDefinition choiceDef = (ExtensibleChoiceParameterDefinition)def;
                 if(choiceDef.getChoiceListProvider() != null)
                 {
-                    choiceDef.getChoiceListProvider().onBuildTriggeredWithValue(job, choiceDef, value.value);
+                    choiceDef.getChoiceListProvider().onBuildTriggeredWithValue(job, choiceDef, value.getValue().toString());

```

Comment:
`StringParameterValue#value` is public, but annotated not to export.
Unfortunately, somehow the type of `StringParameterValue#getValue()` is `Object` and requires casting or `toString()`.
(Really strange!)